mcp/src/main/java/io/modelcontextprotocol/spec/McpClientSession.java

```diff
@@ -221,7 +221,7 @@ private Mono<Void> handleIncomingNotification(McpSchema.JSONRPCNotification noti
 		return Mono.defer(() -> {
 			var handler = notificationHandlers.get(notification.method());
 			if (handler == null) {
-				logger.error("No handler registered for notification method: {}", notification.method());
+				logger.warn("No handler registered for notification: {}", notification);
```

Comment:
Log abnormal behavior without indicating it's an error. 
Perhaps consider using debug level logging?